packages/plugin-conversation/src/share-activity.js

```diff
@@ -316,7 +334,14 @@ ShareActivity.create = function create(conversation, object, spark) {
   });
 
   if (files) {
-    files.forEach((file) => share.add(file));
+    files.forEach((file) => {
+      if (Array.isArray(file)) {
```

Comment:
This happens when and activity share is created from an object.

`spark.conversation.share(conversation, { files: [....] }`